pkg/tms/tmsClient.go

```diff
@@ -200,14 +203,18 @@ func (communicationInstance *CommunicationInstance) UploadFileToNode(nodeName, f
 	}
 
 	json.Unmarshal(data, &nodeUploadResponseEntity)
-	if communicationInstance.isVerbose {
-		communicationInstance.logger.Info("Node upload executed successfully")
-	}
+	communicationInstance.logger.Info("Node upload executed successfully")
+
+	// Important: there are Customers, who might rely on format of this log message to parse transport request id
```

Comment:
And we cannot offer them something else, such as variables in the common pipeline environment (CPE)? Maybe additionally to the log message we could also provide the information in CPE variables. Then they can use that in the pipeline, without having to parse the log.

Reply:
> And we cannot offer them something else, such as variables in the common pipeline environment (CPE)?

Yes, that will be an ideal case. For now, we have agreed with the Customer, who complained about missing log message after we switched to the new Golang implementation of the step, that we provide a log message with transport request id in the Golang implementation as well.

Usage of variables in CPE or other options, if they are, have to be and will be discussed separately.

Reply:
Ok, let's do so. @marcusholl What do you think? Would it help Piper users, if they could retrieve this information from the CPE, rather than parsing the log? Is there something else we could offer, apart from the CPE?